<?xml version="1.0"?>
<launch>
 
  <arg name="cmd_vel_topic" value="/cmd_vel" /> 
  <arg name="pose_topic" value="/ground_truth/state" />


<!-- Run the force field node that find the envirnmental force that acts on the the robot -->
<node name="force_field" pkg="navigation" type="force_field" output="screen">
    <rosparam command="load" file="$(find navigation)/config/force_field_params.yaml" />
    <remap from="/cmd_vel" to="$(arg cmd_vel_topic)"/> 
    <remap from="/RosAria/pose" to="$(arg pose_topic)"/> 
    <remap from="/RosAria/sonar" to="/cloud"/> 
</node>

 

</launch>
